plugin/trino-base-jdbc/pom.xml

```diff
@@ -156,6 +151,12 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>log-manager</artifactId>
+            <scope>provided</scope>
```

Comment:
That's weird. this looks like `<!-- Trino SPI -->` section, but log-manager isn't a SPI dep